write-api-client/pom.xml

```diff
@@ -0,0 +1,259 @@
+<!--
+  ~    Copyright 2016 OICR
+  ~
+  ~    Licensed under the Apache License, Version 2.0 (the "License");
+  ~    you may not use this file except in compliance with the License.
+  ~    You may obtain a copy of the License at
+  ~
+  ~        http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~    Unless required by applicable law or agreed to in writing, software
+  ~    distributed under the License is distributed on an "AS IS" BASIS,
+  ~    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~    See the License for the specific language governing permissions and
+  ~    limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.dockstore</groupId>
+        <artifactId>write-api</artifactId>
+        <version>0.1-alpha.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>write-api-client</artifactId>
+    <version>0.1-alpha.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>write-api-client</name>
+    <description>write api client</description>
+    <url>https://github.com/ga4gh/dockstore-support</url>
+
+    <prerequisites>
+        <maven>3.2.3</maven>
+    </prerequisites>
+
+
+    <properties>
+        <cwlavro.version>1.0.6</cwlavro.version>
+        <dropwizard.version>1.0.5</dropwizard.version>
+        <jackson.version>2.7.8</jackson.version>
+        <jersey.version>2.23.2</jersey.version>
+        <jetty.version>9.3.9.v20160517</jetty.version>
+        <swagger-core-version>1.5.9</swagger-core-version>
+    </properties>
+
+    <organization>
+        <name>Dockstore</name>
+        <url>https://www.dockstore.org/</url>
+    </organization>
+
+    <scm>
+        <connection>${github.url}</connection>
+        <developerConnection>${github.url}</developerConnection>
+        <url>${github.url}</url>
+    </scm>
+
+    <repositories>
+        <repository>
```

Comment:
This should use artifacts.oicr.on.ca in addition now. See Dockstore develop for an example